lib/meilisearch.ex

```diff
@@ -1,4 +1,6 @@
 defmodule Meilisearch do
+  @version "1.1.0"
```

Comment:
Could we use `Application.spec(:meilisearch_ex)[:vsn]` to not have to maintain the version number at multiple places ?

```
❯ iex -S mix                                                                                                                                                                                               (base) 
Erlang/OTP 25 [erts-13.2.2.1] [source] [64-bit] [smp:32:24] [ds:32:24:10] [async-threads:1] [jit:ns]

Interactive Elixir (1.14.3) - press Ctrl+C to exit (type h() ENTER for help)
iex(1)> Application.spec(:meilisearch_ex)[:vsn]
'1.1.0'
```